### PR TITLE
Bump opentelemetry-mapping-go to v0.13.3 and opentelemetry.io dependencies to v0.93.0

### PR DESCRIPTION
**Pull request by liustanley**
### What does this PR do?

Bump dependencies.

### Motivation

Include https://github.com/DataDog/opentelemetry-mapping-go/pull/265

### Describe how to test/QA your changes

1. Run a Java application instrumented with [opentelemetry-java-instrumentation v2.0.0](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v2.0.0) and verify that the default JVM metrics dashboard is being populated with data.

2. Send an OTLP histogram with no buckets with the default OTLP ingest configuration. Check that something comes out on the other side :)

3. Run app from this branch: https://github.com/DataDog/opentelemetry-examples/pull/59, with `docker-compose.jmxmetricsgatherer.yaml` but replace the OTel collector by DD agent with OTLP ingest. Make sure the OOTB Kafka dashboard is being populated with data.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided. Except if the `qa/skip-qa` label, with required either `qa/done` or `qa/no-code-change` labels, are applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by ahmed-mez — APPROVED** [review]
Looks good for trace-agent files.

**Review by mx-psi — APPROVED** [review]

File: releasenotes/notes/otel-mapping-v0.13.3-2b683568f8a3befb.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    OTLP ingest supports stable Java runtime metrics introduced in opentelemetry-java-instrumentation v2.0.0.
+    OTLP ingest supports kafka metrics mapping. This allows users of the JMX Receiver/JMX Metrics Gatherer and kafka metrics receiver to have access to the OOTB kafka Dashboard.
```
Comment (jhgilbert) [inline releasenotes/notes/otel-mapping-v0.13.3-2b683568f8a3befb.yaml:12]:
```suggestion
    OTLP ingest supports Kafka metrics mapping. This allows users of the JMX Receiver/JMX Metrics Gatherer and Kafka metrics receiver to have access to the OOTB Kafka Dashboard.
```

File: releasenotes/notes/otel-mapping-v0.13.3-2b683568f8a3befb.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    OTLP ingest supports stable Java runtime metrics introduced in opentelemetry-java-instrumentation v2.0.0.
```
Comment (jhgilbert) [inline releasenotes/notes/otel-mapping-v0.13.3-2b683568f8a3befb.yaml:11]:
```suggestion
    OTLP ingest supports stable Java runtime metrics introduced in `opentelemetry-java-instrumentation` v2.0.0.
```

File: releasenotes/notes/otel-mapping-v0.13.3-2b683568f8a3befb.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    OTLP ingest supports kafka metrics mapping. This allows users of the JMX Receiver/JMX Metrics Gatherer and kafka metrics receiver to have access to the OOTB kafka Dashboard.
+fixes:
+  - |
+    Fixes a bug in OTLP ingest where empty histograms were not being sent to the backend in the distributions mode. Empty histograms are now mapped as if they had a single (min, max) bucket.
```
Comment (jhgilbert) [inline releasenotes/notes/otel-mapping-v0.13.3-2b683568f8a3befb.yaml:15]:
```suggestion
    Fixes a bug in OTLP ingest where empty histograms were not being sent to the backend in the distributions mode. Empty histograms are now mapped as if they had a single `(min, max)` bucket.
```

**Review by jhgilbert — APPROVED** [review]
Approved with minor suggestions for style. Thank you!

**Review by rayz — APPROVED** [review]

**Review by mackjmr — APPROVED** [review]

**Review by jeremy-hanna — APPROVED** [review]

**Pull request merged**